src/libopensc/asn1.c

```diff
@@ -1689,7 +1689,7 @@ static int asn1_decode(sc_context_t *ctx, struct sc_asn1_entry *asn1,
 	size_t left = len, objlen;
 
 	sc_debug(ctx, SC_LOG_DEBUG_ASN1,
-		 "%*.*scalled, left=%"SC_FORMAT_LEN_SIZE_T"u, depth %d%s\n",
+		 "%*.*s called, left=%"SC_FORMAT_LEN_SIZE_T"u, depth %d%s\n",
```

Comment:
Not sure about this fix. Can you have a look?

Reply:
afaik this works as expected, adding indentation based on the depth we are in. Adding one more space indents the whole log message even more. I think the right solution here would be to split these words with the quotes so codespell is happy but the log message is not modified such as
```
-		 "%*.*scalled, left=%"SC_FORMAT_LEN_SIZE_T"u, depth %d%s\n",
+		 "%*.*s""called, left=%"SC_FORMAT_LEN_SIZE_T"u, depth %d%s\n",
```

Reply:
Fixed as suggested.